distro/Makefile

```diff
@@ -29,13 +30,23 @@ get-artifacts:
 	# get artifacts
 	cd .. && mvn clean package && cd -
 	mkdir -p ${DIST}
-	wget -O ${DIST}/${HZ_INSTALL_TAR} "https://download.hazelcast.com/download.jsp?version=hazelcast-${HZ_VERSION}&type=tar&p="
+	mvn dependency:get \
+			-DrepoUrl=https://oss.sonatype.org/content/repositories/snapshots/ \
+        	-Dartifact=com.hazelcast:${HZ_DIST}:${HZ_VERSION}:tar.gz \
+            -Dtransitive=false \
+            -Ddest=${HZ_DIST}-${HZ_VERSION}.tar.gz
+	mv ${HZ_DIST}-${HZ_VERSION}.tar.gz ${DIST}/${HZ_INSTALL_TAR}
 	tar -zxf ${DIST}/${HZ_INSTALL_TAR} -C ${DIST}
+	# in snapshot 5.0 we do not have hazelcast-all-5.0-SNAPSHOT.jar file,
+	# but there is hazelcast-5.0-SNAPSHOT.jar. This creates problem where script(s)
+	# such as bin/hz set classpath as hazelcast-all-5.0-SNAPSHOT.jar, where it should actually
+	# be the other one
+	# todo: this should be either generalized here, or get fixed in hz release
+	mv ${DIST}/${HZ_INSTALL_NAME}/lib/hazelcast-${HZ_VERSION}.jar ${DIST}/${HZ_INSTALL_NAME}/lib/hazelcast-all-${HZ_VERSION}.jar;
```

Comment:
We are removing hazelcast-all in 5.0, so the scripts should be updated to use hazelcast-${HZ_VERSION}.jar.